pages/contributors.tsx

```diff
@@ -73,83 +76,286 @@ const ContributorsPage: FC<{ contributors: Contributor[] }> = ({
+                      <FaGithub className="text-2xl" />{' '}
+                    </div>
+                    GitHub
+                  </Link>
+                </div>
+                {contributor.twitter_username && (
+                  <div className="mt-1 hover:bg-[#9F87FF1A] hover:text-gray-600 dark:hover:text-violet-300 py-1 px-4 mr-2 rounded-md transition-all duration-300 ease-in-out text-sm">
+                    <Link
+                      href={`https://twitter.com/${contributor.twitter_username}`}
+                      aria-label={`Twitter Profile of ${contributor.twitter_username}`}
+                      className="flex flex-col items-center justify-center"
+                      {...linkProps}
+                    >
+                      <div className="pb-2">
+                        {/* <div className="bg-black rounded-full p-2"> */}
+                        <FaXTwitter className="text-2xl" /> {/* </div> */}
+                      </div>
+                      Twitter
+                    </Link>
+                  </div>
+                )}
               </div>
             </div>
-            <div className="flex justify-evenly mt-auto">
-              <Link
-                href={`https://github.com/${contributor.login}`}
-                {...linkProps}
-                aria-label={`GitHub Profile of ${contributor.login}`}
-              >
-                GitHub
-              </Link>
-              <Link
-                href={`https://github.com/rupali-codes/LinksHub/commits?author=${contributor.login}`}
-                {...linkProps}
-                aria-label={`Commit History of ${contributor.login} in LinksHub`}
-              >
-                Contributions
-              </Link>
+          ))}
+      </div>
+      <div className="bg-white dark:bg-[#293242] flex justify-between flex-col md:flex-row rounded-xl py-6 px-4 my-10">
+        <div className="flex space-x-4">
+          <div className="pl-2 pr-3">
+            <FaTrophy className="text-2xl text-[#FBD449]" />{' '}
+          </div>
+          <div className="flex flex-col">
+            <div className="text-[#293242] dark:text-white text-lg pb-1">
+              Join our awesome team!
+            </div>
+            <div className="text-gray-400 pr-2 md:pr-4">
+              Be a contributor and improve LinksHub and help fellow developers.
             </div>
           </div>
-        ))}
+        </div>
+        <div className="bg-[#714EFF] hover:bg-[#7560ca] rounded-2xl h-14 w-40 px-6 text-white mr-2 whitespace-nowrap mt-6 ml-[56px] md:mt-0 md:ml-0 duration-300 transition-all">
+          <Link
+            href={'https://github.com/rupali-codes/LinksHub'}
+            aria-label="GitHub Link to LinksHub"
+            {...linkProps}
+          >
+            <div className="flex items-center justify-center py-4 text-md">
+              Join us now
+              <span className="pl-2 text-sm">
+                {' '}
```

Comment:
Instead of the space, what do you think of adding a gap class on the parent element?

Reply:
It is because of prettier, and it will work even without space due to tailwind styling. And LinksHub is based on prettier.

![image](https://github.com/rupali-codes/LinksHub/assets/74038190/fbbb3068-fbec-4159-a0fd-c9bcd483b686)